views/products/presentation/disco.tpl

```diff
@@ -0,0 +1,22 @@
+<p align="center">
+  <img class="dark" style="width: 60%" src="/resources/products/images/disco/logo.svg" />
+</p>
+
+<p>
+  Disco greatly simplify access to
+  <a href="https://en.wikipedia.org/wiki/Distributed_computing">distributed</a>
+  <a href="https://en.wikipedia.org/wiki/Machine_learning">machine learning</a>,
+  It runs in a browser, allowing it to work on pretty much any platform, simply
+  open the link in a browser and you're using it.
+  <br>
+  It allows creating your own machine learning tasks so that others can help
+  with theses by just loading their data in the browser.
+  By computing theses tasks, you create a machine learning model, a mathematical
+  system which allows a program to make predictions based on a given set of data.
+  It fits many use cases, such as determining the likeliness that some hospital
+  patient has a disease, or automatically determine what an image represents.
+</p>
```

Comment:
Copy the list of the papers Disco (will be / is) based on from the github page:

https://github.com/epfml/disco/blame/develop/README.md#L21

And describe which algorithms are already implemented, and which will (probably) be.

Reply:
Papers added